config/default2.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@ranjit977 remove this file

Reply:
I have removed it from my local. Can you remove it?